packaging/linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,68 @@
+name: sunshine
+base: core22
+version: '0.14.1'
+summary: Sunshine is a Gamestream host for Moonlight.
+description: |
+  Sunshine is a Game stream host for Moonlight. Sunshine is a self hosted, low latency, cloud gaming solution with support for AMD, Intel, and Nvidia gpus. It is an open source implementation of NVIDIA's GameStream, as used by the NVIDIA Shield. Connect to Sunshine from any Moonlight client, available for nearly any device imaginable.
+
+grade: devel
+confinement: devmode
+
+layout:
+  /usr/local/sunshine:
+    bind: $SNAP/usr/local/sunshine
```

Comment:
if we change to `-DSUNSHINE_ASSETS_DIR='share/sunshine`, should this bind path also update?